src/sc_generation/abi_wasmv1_generation/bytes_to_base58_check.rs

```diff
@@ -4,8 +4,18 @@ use super::generate_string;
 
 pub fn generate_abi_bytes_to_base58_check(calls: &mut Vec<String>) {
     let mut rng = rand::thread_rng();
+    let s = generate_string(rng.gen_range(2..500));
```

Comment:
I'd replace with 
```rust
let s = generate_string(rng.gen_range(2..500)).as_bytes().to_vec();
```
and drop the line with unsafe block
as generate_string returns an ascii string I think we can get ride of all the utf16 stuff

Reply:
I'll update my helper function to do that, yes :).
Note: in that case, we have to use a gen_range of 5..1000 to keep the same number of bytes

Reply:
Instead I just generate random bytes directly, it makes more sense to me.
Please don't hesitate to comment further on some improvements related to this!

Reply:
yes you can increase the length of the string, that's an important point.
What we don't care is how we generate the [u8] (we could even have a generate_vec_u8() here)